tox.ini

```diff
@@ -24,6 +24,7 @@ norecursedirs = .* docs requirements
 
 [testenv]
 deps =
+    django18: Django==1.8.18
```

Comment:
Nit: I'd probably write this as `Django>=1.8,<1.9`

Reply:
Yeah.. I thought I'd pin it to the Ginkgo revision, but I see your point..  Fixed with 68361d8a824cca9683665374648b7c456f2eb3c4